src/presentation/pages/Note.vue

```diff
@@ -26,6 +26,7 @@
       :read-only="!canEdit"
       @change="noteChanged"
     />
+    <div id="note-clone" />
```

Comment:
seems unexpected, it's better to incapsulate all screenshot logic at the util level

Reply:
removed